sqlglot/parser.py

```diff
@@ -173,10 +173,13 @@ class Parser:
         TokenType.FORMAT,
         TokenType.FUNCTION,
         TokenType.IF,
+        TokenType.ISNULL,
         TokenType.INTERVAL,
         TokenType.LAZY,
         TokenType.LOCATION,
         TokenType.NEXT,
+        TokenType.NULLS_FIRST,
```

Comment:
these are not needed in id_var_tokens because they have spaces, only the ones that could become aliases need to go in here